### PR TITLE
86dueakk7: Create hero

### PR DESCRIPTION
**Pull request by lualreye**
The client should see a stunning hero to them scroll to the next section
The section should look like the image attached
The content 'copy' is provided by  
Keep working on the Hero componente already created on main folder in components

Some changes added to the content

### CODE REVIEW

File: components/Main/Hero.vue [hunk 1]
```diff
@@ -1,17 +1,189 @@
 <script setup lang="ts">
+import { onMounted, onUnmounted } from 'vue';
```
Comment (yeyodev1) [inline components/Main/Hero.vue:2]: nuxt 3 uses a system names 'auto-imports' so for that is unnecesary import that type of things, i'll leave you the documentation anyway [nuxt](https://nuxt.com/docs/guide/concepts/auto-imports)

File: style/_fontVariables.scss
Comment (yeyodev1) [inline style/_fontVariables.scss:1]: let me know in what cases we use the new font that you added

**Review by yeyodev1 — CHANGES_REQUESTED** [review]

File: components/Main/Hero.vue
Comment (yeyodev1) [inline components/Main/Hero.vue:1]: Captura de pantalla 2024-08-20 a la(s) 8 37 32 p  m


I think the view has to has more height, at least 70% ot 60%, only that looks good

**Review by yeyodev1 — CHANGES_REQUESTED** [review]

File: style/_fontVariables.scss
Comment (lualreye) [reply style/_fontVariables.scss:1]: just for the title as far as I can see

File: components/Main/Hero.vue (hunk 1 above)
Comment (lualreye) [reply components/Main/Hero.vue:2]: hahahahahhahahahhahahhahahahhahahhahahahha ok I'll read it thanks

**Review by yeyodev1 — APPROVED** [review]
looks awesome

**Pull request merged**